pkg/cli/credential.go

```diff
@@ -0,0 +1,79 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/acorn-io/cmd"
+	"github.com/gptscript-ai/gptscript/pkg/config"
+	"github.com/gptscript-ai/gptscript/pkg/credentials"
+	"github.com/spf13/cobra"
+)
+
+type Credential struct {
+	root        *GPTScript
+	AllContexts bool `short:"A" usage:"List credentials for all contexts" local:"true"`
```

Comment:
Don't do `-A`.  Either have no short or if you really want a short value then `-a`.  The `-A` is very kubernetes-y.

Reply:
Fixed